### ISSUE TITLE
name and title subject headings should be concatenated

### ISSUE THREAD

**Issue by theLinkResolver** [ualbertalib/discovery#1489]
**Describe the bug**
Some elements of name and title subject headings must logically go together but are currently being displayed as separate searchable elements.

**To Reproduce**
https://search.library.ualberta.ca/catalog/6784731
- The text "Elizabeth II, Queen of Great Britain, 1926-" should not be broken into parts

https://search.library.ualberta.ca/catalog/824831
- The text "Olympic Winter Games (15th : 1988 : Calgary, Alta.)" should not be broken into parts

**Expected behavior**
Certain MARC subfields should be treated as indivisible strings:
600abcdq
610abcdfk
611acdn
630adlnps

We can provide more information as needed.

**Comment by seanluyk** [comment]
Thanks for documenting this, @theLinkResolver. This applies to NEOS BL as well. It looks like the logic to split subject headings isn't sensitive to these differences. Adding in the first example from [Stanford](https://searchworks.stanford.edu/view/9438409), and [Cornell](https://newcatalog.library.cornell.edu/catalog/9709768). If they've figured out how to deal with this we should borrow their approach

**Comment by theLinkResolver** [comment]
@seanluyk Agreed - Brian and I did some looking at Stanford and we see evidence that they gave thoughtful consideration to how terms are joined/split. 

Also, my apologies that I never noticed these issues when the feature was originally developed - that is on me entirely.

**Comment by seanluyk** [comment]
@theLinkResolver no need to apologize, we all missed it! I think the good news is that if a user clicks the last string in a series, the search is still correct.
